src/v2_services/compare_service.py

```
@@ -0,0 +1,341 @@
+    # LLM이 가끔 JSON 형식을 지키지 않아 파싱에 실패할 수 있습니다.
+    # 이 경우 에러를 발생시키지 않고, 기본값("분석 실패")으로 대체합니다.
+    try:
+        analysis = await chain.ainvoke(
+            {
+                "context_1": context_1,
+                "context_2": context_2,
+                "question": query,
+                "format_instructions": format_instructions,
+            }
+        )
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)